my-app/src/Authentication.tsx

```diff
@@ -0,0 +1,56 @@
+import { AUTH_TOKEN } from './constants';
+import gql from 'graphql-tag';
+import { ApolloClient } from 'apollo-client';
+import { createHttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+
+
+const LOGIN_MUTATION = gql`
+mutation LoginMutation($email: String!, $password: String!){
+  login(data:{email: $email, password: $password}){
+    token
+  }
+}
+`
+
+const httpLink = createHttpLink({
+  uri: 'https://tq-template-server-sample.herokuapp.com/graphql'
+})
+
+const client = new ApolloClient({
+  link: httpLink,
+  cache: new InMemoryCache()
+})
+
+
+class Authentication  {
+    constructor(){
+    }
+    public result:any;
+    login(email, password) {
```

Comment:
Na formatacão de uma classe, acho que faz sentido você pular linha entre os métodos. Outra coisa que pode ajudar é deixar todas as declarações no início (antes do construtor), porque podem ter variáveis que são inicializadas nele.